fw/token_tables.h

```diff
@@ -0,0 +1,236 @@
+/**
+ *		Tempesta FW
+ *
+ * Shared symbol tables for different classes of tokens.
+ *
+ * Copyright (C) 2016-2023 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+/*
+ * ASCII codes to accept URI string.
+ */
+#define TOKEN_TBL_URI	0
+#define TOKEN_SYM_URI   \
```

Comment:
We aling `\` in macros:
```suggestion
#define TOKEN_SYM_URI                                    \
```
Please fix here and in all the table below